### PR TITLE
Update credits section on github page

### LINKED ISSUE #117: Update credits section of GitHub Page
Here's where you can find the outdated info https://github.com/wlsf82/protractor-helper/blob/master/index.md#credits.

And here's the updated one https://github.com/wlsf82/protractor-helper#credits.

### PR DESCRIPTION
**Pull request by pedrohyvo**
Now, README and github page are accurate. Closes #117

## Pull Request Checklist

- [x] Have you followed the steps in our [Contributing](https://github.com/wlsf82/protractor-helper/blob/master/docs/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wlsf82/protractor-helper/pulls) for the same update/change?
- [x] Have you successfully ran tests with your changes locally?
- [x] Have you written new tests for your changes, if necessary?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you made improvements to the documentation, if necessary? ([docs/](https://github.com/wlsf82/protractor-helper/tree/master/docs) folder and [README.md](https://github.com/wlsf82/protractor-helper/blob/master/README.md))
- [x] Have you linked the correct [GitHub issue](https://github.com/wlsf82/protractor-helper/issues), if there is any?

> Pull requests that do not pass the coding static analysis and the automated tests on [Github Actions](https://github.com/wlsf82/protractor-helper/actions), and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed.

### CODE REVIEW

**Review by wlsf82 — CHANGES_REQUESTED** [review]
@pedrohyvo, it seems that your PR is behind master by one commit.
Can you rebase it based on the original repository so that only the `index.md` file will be changed?

**Comment by pedrohyvo** [comment]
@wlsf82 now it's ready to go 🚀

File: index.md [hunk 1]
```diff
@@ -23,10 +23,15 @@ const protractorHelper = require("protractor-helper");
...
 
-The **protractor-helper** library was created by [Walmyr Filho](https://br.linkedin.com/in/walmyr-lima-e-silva-filho-147a9110a) and is kept together with [Paulo Gonçalves](https://www.linkedin.com/in/paulo-goncalves/).
+The **protractor-helper** library was created and is maintained by [Walmyr Filho](https://br.linkedin.com/in/walmyr-lima-e-silva-filho-147a9110a).
+
+Follow Walmyr on Twitter ([@wlsf82](https://twitter.com/walmyrlimaesilv)).
+
+Thanks to [Paulo Gonçalves](https://www.linkedin.com/in/paulo-goncalves/), [Lucas Amaral](https://www.linkedin.com/in/lopesdoamaral/), [Michiel Cuijpers](https://github.com/MichielCuijpers), [Ghazi Khan](https://codewithghazi.com/), and [Pedro Hyvo](https://www.linkedin.com/in/pedrohyvo/) for contributing to the project.
 
----
+---
```
Comment (wlsf82) [inline index.md:35]:
```suggestion
---
```

Comment (wlsf82) [inline index.md:35]: Oops, it seems that you add a space by mistake.

**Review by wlsf82 — CHANGES_REQUESTED** [review]
Nitpick.

File: index.md (hunk 1 above)
Comment (wlsf82) [reply index.md:35]: Or is it on purpose? Since the line looks thicker here https://github.com/pedrohyvo/protractor-helper/blob/master/index.md

**Review by wlsf82 — APPROVED** [review]
Thanks for another contribution, @pedrohyvo! 👊🏻

File: index.md [hunk 2]
```diff
@@ -23,9 +23,14 @@ const protractorHelper = require("protractor-helper");
...
 [Here you'll find examples of usage of each of the available helper functions.](https://www.npmjs.com/package/protractor-helper#available-helpers)
+
 ## Credits
 
-The **protractor-helper** library was created by [Walmyr Filho](https://br.linkedin.com/in/walmyr-lima-e-silva-filho-147a9110a) and is kept together with [Paulo Gonçalves](https://www.linkedin.com/in/paulo-goncalves/).
+The **protractor-helper** library was created and is maintained by [Walmyr Filho](https://br.linkedin.com/in/walmyr-lima-e-silva-filho-147a9110a).
+
+Follow Walmyr on Twitter ([@wlsf82](https://twitter.com/walmyrlimaesilv)).
+
+Thanks to [Paulo Gonçalves](https://www.linkedin.com/in/paulo-goncalves/), [Lucas Amaral](https://www.linkedin.com/in/lopesdoamaral/), [Michiel Cuijpers](https://github.com/MichielCuijpers), [Ghazi Khan](https://codewithghazi.com/), and [Pedro Hyvo](https://www.linkedin.com/in/pedrohyvo/) for contributing to the project.
```
Comment (PauloGoncalvesBH) [inline index.md:33]:
```suggestion
Thanks to [Paulo Gonçalves](https://www.linkedin.com/in/paulo-goncalves/), [Lucas Amaral](https://www.linkedin.com/in/lopesdoamaral/), [Ghazi Khan](https://codewithghazi.com/), and [Pedro Hyvo](https://www.linkedin.com/in/pedrohyvo/) for contributing to the project.
```

@wlsf82 será necessário a inclusão do Michiel Cuijpers? Sua contribuição foi a inclusão do badge do Better Code Hub, que ele tem alguma ligação direta.

https://github.com/wlsf82/protractor-helper/commits?author=MichielCuijpers

Comment (wlsf82) [reply index.md:33]: Toda contribuição é contribuição @PauloGoncalvesBH, não vejo porquê excluí-lo.

**Review by wlsf82 — CHANGES_REQUESTED** [review]
@pedrohyvo, it seems that your branch is behind the main repo again. Besides this PR commits, I also see the one from a previous PR, where you added the MIT license badge. Can you rebase it, please?

**Comment by pedrohyvo** [comment]
> @pedrohyvo, it seems that your branch is behind the main repo again. Besides this PR commits, I also see the one from a previous PR, where you added the MIT license badge. Can you rebase it, please?

Removed @wlsf82

**Review by PauloGoncalvesBH — APPROVED** [review]

**Review by wlsf82 — CHANGES_REQUESTED** [review]
@pedrohyvo (and @PauloGoncalvesBH), your PR should be changing only the `index.md` file, but instead, it's also changing the `README.md` and `docs/PT-BR/README.md` files.

I suggest that you reset your branch based on the `master` branch of the main repo and then add only the commit that changes what needs to be changed, and finally, force a push.

**Comment by wlsf82** [comment]
@pedrohyvo, to avoid more going back and forth, I'll close this PR without merging it, and I'll create one myself and add you as a reviewer, ok?

**Pull request closed without merging**